content/blog/2023/04/26/research-community-open-call.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Research Community Open Call #1 – April 2023"
+created: 2023-04-26
+categories:
+  - commmunity-call
+authors:
+  - lifexitself
+---
+
+This is the first in a series of monthly open meetings hosted by the Life Itself research community. These purpose of these calls is to provide an opportunity for newcomers to learn about the community and engage in discussions, and for community members to share updates about ongoing work and coordinate across interest groups.
+
+In this opening call, Rufus proposes an initial framework for a research agenda aligned with Life Itself’s wider purpose of paradigmatic transition. This approach is informed by the [[primacy-of-being|primacy of being]] thesis and is primarily focused on the topics of inner development and cultural evolution, with a secondary focus on systems transformation and the politics of paradigmatic change.
+
+![[excalidraw/research-topical-map-2023-01-14.excalidraw.svg]]
+
+> You can find more detailed information about the research community—including our approach, focus topics and ongoing projects—on our [[research|overview page]].
+
+### How to participate
```

Comment:
Formatting: normally spacing around headings, paragraphs etc - see https://www.datopian.com/playbook/style-guide/markdown if you want to get picky 😉